Wiredcraft/Macros/WCConstants.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  WCConstants.swift
+//  Wiredcraft
+//
+//  Created by codeLocker on 2020/6/10.
+//  Copyright © 2020 codeLocker. All rights reserved.
+//
+
+import UIKit
+
+/*
+ WCConstants is used to define constant data that is common in some projects
+ such as color、font、hint .etc
+ */
+public struct WCConstants {
```

Comment:
Hey @codeLockers, thank you for the PR. I'm going to ask you few questions about your solution.

Here and in other places where you used `public` access control for the class, I'd like to know why you choose it (public)?

Reply:
hey @illuminati1911  thank you for the review.
There's no need to use public here actually, The main reason for using public is personal habits.
i did a project once, in that project,  We split the project into several modules, the main project use cocoapods to group each module.
so when i meet some class or function that may be used by other modules, i use public habitually.
but in different project, we need to use the most appropriate access  control